webapp/channels/src/components/latex_inline/latex_inline.test.tsx

```diff
@@ -24,19 +27,21 @@ describe('components/LatexBlock', () => {
             enableInlineLatex: false,
         };
 
-        const wrapper = shallow(<LatexInline {...props}/>);
-        await import('katex'); //manually import katex
-        expect(wrapper).toMatchSnapshot();
+        render(<LatexInline {...props}/>);
+        const wrapper = await screen.findAllByTestId('latex-disabled');
+        expect(wrapper.length).toBe(1);
+        expect(wrapper.at(0)).toMatchSnapshot();
     });
 
     test('error in katex', async () => {
         const props = {
-            content: 'e^{i\\pi + 1 = 0',
+            content: '```latex e^{i\\pi + 1 = 0```',
             enableInlineLatex: true,
         };
 
-        const wrapper = shallow(<LatexInline {...props}/>);
-        await import('katex'); //manually import katex
-        expect(wrapper).toMatchSnapshot();
+        render(withIntl(<LatexInline {...props}/>));
+        const wrapper = await screen.findAllByTestId('latex-enabled');
+        expect(wrapper.length).toBe(1);
+        expect(wrapper.at(0)).toMatchSnapshot();
```

Comment:
`render` should return, among other things, the container. So you can do something like this:
```suggestion
        const {container} = render(withIntl(<LatexInline {...props}/>));
        expect(container).toMatchSnapshot();
```

Reply:
Thanks for the suggestion, I have used this logic to update the test and also used 'act' to handle the state changes.